src/service/medicalRecordService.js

```
@@ -0,0 +1,14 @@
+import axiosInstance from "./axiosInstance";
+
+class MedicalRecordService {
+  static async getAllRecords() {
+    try {
+      const response = await axiosInstance.get(`/record/getall`);
+      return response.data;
+    } catch (error) {
+      throw error;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)